### ISSUE TITLE
ICICI bank home page! Next button is not label properly 

### ISSUE THREAD

**Issue by nsingh2001** [nsingh2001/bicici#6]
UAT 2022-04
Windows 10
Chrome Version 100.0.4896.88 (Official Build) (64-bit)

NVDA 2021.3.1
Repro Steps
1. Open the URL  https://www.icicibank.com/
2.after tab 21 time there is Next button 

Expected Result the Next button should be announce the purpose properly

Actual Result screen reader announce only Next button 
WCAG Guideline
1. https://www.w3.org/TR/WCAG21/#name-role-value
